build_win32.cmd

```diff
@@ -38,6 +39,6 @@ rename cod4u_win32.exe _____________________________________________cod4u_win32.
 dlltool -D _____________________________________________cod4u_win32.exe -d cod4u_win32.def -l ..\plugins\libcom_plugin.a
 rename _____________________________________________cod4u_win32.exe cod4u_win32.exe
 
-echo Done!
+echo "Done! File is in %currentDir%/bin
```

Comment:
```suggestion
echo Done! File is in %currentDir%/bin
```
Can get rid of the quotes, batch will just output them directly to the console otherwise. 😉

Reply:
I left a quote out by mistake anyway, didn't know that about batch tho!